Examples/Scripts/MaterialMapping/MaterialComposition.cpp

```diff
@@ -15,19 +17,6 @@
 #include <TApplication.h>
 #include <boost/program_options.hpp>
 
-#define BOOST_AVAILABLE 1
-#if ((BOOST_VERSION / 100) % 1000) <= 71
-// Boost <=1.71 and lower do not have progress_display.hpp as a replacement yet
-#include <boost/progress.hpp>
-using progress_display = boost::progress_display;
-#else
-// Boost >=1.72 can use this as a replacement
-#include <boost/timer/progress_display.hpp>
-using progress_display = boost::timer::progress_display;
-#endif
-
-#include "materialComposition.C"
```

Comment:
This include conditionally uses `progress_display`. What's the intention of removing this? If we indeed want to remove it, will you remove it from `materialComposition.C` as well?